packages/front/src/routes/association/[identifier]/Association.controller.test.js

```diff
@@ -38,4 +43,28 @@ describe("Association Controller", () => {
             });
         });
     });
+
+    describe("getRnaSirenDuplicates()", () => {
+        const RNA_SIREN_2 = { rna: RNA_SIREN.rna, siren: "SIREN_2", id: "QNCQZ9822N49" };
+        const RNA_SIREN_3 = { rna: "RNA_2", siren: RNA_SIREN.siren, id: "MZD92Z0jj0àç" };
+        const mockGetAssociatedIdentifier = vi.fn(async () => [RNA_SIREN]);
+        beforeAll(() => mockedRnaSirenService.getAssociatedIdentifier.mockImplementation(mockGetAssociatedIdentifier));
+
+        it("should set duplicates from RNA", async () => {
+            mockedRnaSirenService.getAssociatedIdentifier.mockImplementationOnce(async () => [RNA_SIREN, RNA_SIREN_2]);
+            controller.getRnaSirenDuplicates("RNA", "SIREN");
+            // wait for async mocked function to resolve
+            await new Promise(process.nextTick);
```

Comment:
pourquoi créer une nouvelle promesse plutôt que 
```suggestion
            await process.nextTick();
```

Reply:
simple copier coller d'une solution, je vérifie que ça fonctionne bien de la même manière

Reply:
ici une réponse : https://stackoverflow.com/questions/76980557/why-is-process-nexttick-wrapped-in-a-promise-in-this-statement

Reply:
c'est surtout un hack pour s'assurer que toutes exécutions en cours avant la création de la promesse soit terminées (enfin si j'ai bien compris)

Et `process.nextTick()` n'est pas valable car s'attend à avoir un callback en paramètre (donc erreur TS mais on pourrait by-pass ça)

Reply:
ok je vois ça me va. J'ai appris un truc :)

Reply:
J'ai pas saisi à 100% le pourquoi du comment ceci dit